nf_core/bump_version.py

```diff
@@ -63,7 +88,7 @@ def update_file_version(filename, lint_obj, pattern, newstr):
     matches = re.findall(pattern, content)
     if len(matches) == 0:
         raise SyntaxError ("Could not find version number in {}: '{}'".format(filename, pattern))
-    if len(matches) > 1:
+    if len(matches) > 1 and not allow_multiple:
```

Comment:
What's the reasoning here?

Reply:
Previously I had it fail if it found more than one occurence, as I was being cautious and I figured that it could mean a dodgy regex or something unexpected in the target file. However, in the `.travis.yml` file it's entirely possible to define the minimum nextflow version a few times (see for example the [methylseq pipeline](https://github.com/nf-core/methylseq/blob/93600734bc7a1c1a12a3848b4429425ee3aee79c/.travis.yml#L32-L36)). So this option allows that behaviour for just that file and that version change.